nano/core_test/node.cpp

```diff
@@ -4300,161 +4300,176 @@ TEST (rep_crawler, local)
 
 TEST (node, pruning_automatic)
```

Comment:
Can we add a comment describing the intention of this unit test?

Reply:
Will do, thanks